src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/NotifyClaimCallbackHandler.java

```diff
@@ -100,4 +155,29 @@ private SubmittedCallbackResponse buildConfirmation(CallbackParams callbackParam
             .confirmationBody(body)
             .build();
     }
+
+    private SubmittedCallbackResponse buildConfirmationWithSolicitorOptions(CallbackParams callbackParams) {
+        CaseData caseData = callbackParams.getCaseData();
+        String formattedDeadline = formatLocalDateTime(caseData.getClaimDetailsNotificationDeadline(), DATE_TIME_AT);
+        String body;
+
+        if (notifyBothRespondentSolicitors(caseData)) {
+            body = format(CONFIRMATION_SUMMARY, formattedDeadline) + exitSurveyContentService.applicantSurvey();
```

Comment:
Is the logic correct here in the context of backward compatibility?

We need the confirmation screen to remain unchanged until we release multi-party later down the road. This means we need the {body} to always be {CONFIRMATION_SUMMARY} until multi-party feature flag is turned on.

Reply:
Added the following logic for using existing `buildConfirmation` method until multiparty toggle is enabled:
```        
if (!featureToggleService.isMultipartyEnabled()
            || caseData.getDefendantSolicitorNotifyClaimOptions() == null) {
            return buildConfirmation(callbackParams);
        }
```